recipe/meta.yaml

```diff
@@ -1,33 +1,36 @@
 {% set name = "scikit-optimize" %}
-{% set version = "0.3" %}
-{% set bundle = "tar.gz" %}
-{% set hash_type = "sha256" %}
-{% set hash = "d8934c33e5958227d2ff97a4fae2134d2a00432b99e7f9d90b17e45cb1da9bb5" %}
-{% set build = 0 %}
+{% set version = "0.4" %}
+{% set sha256 = "c01d5b9f31105172067b1df18d15c277a15c81580317c2b002d8e63df6216156" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.{{ bundle }}
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  {{ hash_type }}: {{ hash }}
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
 
 build:
-  number: {{ build }}
+  number: 0
   script: python setup.py install --single-version-externally-managed --record=record.txt
 
 requirements:
   build:
     - python
-    - setuptools
+    - cython
```

Comment:
These shouldn't be needed at build time, because of `--single-version-externally-managed`; just `python` and `setuptools` should be enough.

Reply:
@dougalsutherland When I tried that way on my local machine, I get this error:
```
RuntimeError: Setuptools downloading is disabled in conda build. Be sure to add all dependencies in the meta.yaml  url=https://pypi.python.org/simple/matplotlib/
```

Reply:
I just tested and it worked for me....